console/src/tasks.rs

```diff
@@ -184,6 +195,30 @@ impl State {
         }
     }
 
+    pub(crate) fn get_task_details_ref(&self) -> DetailsRef {
```

Comment:
i would probably have just called this
```suggestion
    pub(crate) fn task_details_ref(&self) -> DetailsRef {
```
or even
```suggestion
    pub(crate) fn details_ref(&self) -> DetailsRef {
```
in general, it's [considered more idiomatic](https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter) to only use the `get_` prefix for functions that do some kind of data structure lookup, and avoid it for field accessors like this method.

Reply:
This round of suggestions is also taken care of. Thank you for the reviews!